MovieApp/MovieApp/NewsFeedTableViewController.m

```diff
@@ -0,0 +1,105 @@
+//
+//  NewsFeedTableViewController.m
+//  MovieApp
+//
+//  Created by user on 22/09/16.
+//  Copyright © 2016 internshipABH. All rights reserved.
+//
+
+#import "NewsFeedTableViewController.h"
+#import "FeedTableViewCell.h"
+#import "NewFeedsItem.h"
+#import "FeedDownloader.h"
+
+@interface NewsFeedTableViewController (){
+    NSMutableArray *news;
+    UISearchBar *searchBar;
+    UIBarButtonItem *leftButton;
+    UIBarButtonItem *rightButton;
+}
+
+@end
+
+@implementation NewsFeedTableViewController
+
++(NSURL *)getFeedsSource
+{
+    return [NSURL URLWithString:@"http://www.boxofficemojo.com/data/rss.php?file=topstories.xml"];
+}
+
+-(void)updateViewWithNewData:(NSMutableArray *)feedItemsArray{
+    news=feedItemsArray;
+    
+    if(![news count])
+    {
+        if(!news)
+        {
+            news = [[NSMutableArray alloc] init];
+        }
+        [news addObject:[[NewFeedsItem alloc] initWithHeadline:@"No results" text:@"" sourceUrlPath:@""]];
```

Comment:
Or you can have special view for results case in order to make it more cusomizable.